### ISSUE TITLE
Identifier has already been declared

### ISSUE THREAD

**Issue by sirenkovladd** [putoutjs/minify#24]
- **Version** (`npm -v`): 10.5.2
- **Node Version** `node -v`: v20.13.1
- **OS** (`uname -a` on Linux): Darwin arm64

Identifier has already been declared

```js
import { minify } from "@putout/minify";

const source = `function workLoop() {}
function flushWork() {
	return workLoop();
}
function requestCallback() {
	return flushWork;
}
const sharedConfig = { context: void 0, registry: void 0 };
function setHydrateContext(e) {
	sharedConfig.context = e;
}
var Owner = null;
function createSignal(e) {
	return [e, Object.assign({}).equals];
}
function startTransition() {
	return Owner;
}
const [transPending] = createSignal(!1);
function useTransition() {
	return [transPending, startTransition];
}
function runUserEffects(e) {
	setHydrateContext();
}
global.useTransition = useTransition`;

eval(source);
const result = minify(source, {});
console.log(result);
eval(result);

```

```sh
❯ jq '.version' node_modules/@putout/minify/package.json 
"3.19.7"
❯ node test.js                                                   
const{assign:a}=Object,d=null;function b(){}var[g]=A(!1);function A(e){return[e,a({}).equals]}function f(){return d}const[g]=A(!1);function h(){return[g,f]}global.useTransition=h;
undefined:1
const{assign:a}=Object,d=null;function b(){}var[g]=A(!1);function A(e){return[e,a({}).equals]}function f(){return d}const[g]=A(!1);function h(){return[g,f]}global.useTransition=h;
                                                                                                                          ^

SyntaxError: Identifier 'g' has already been declared

Node.js v20.13.1
```

**Comment by coderaiser** [comment]
Thank you! Fixed in [`v3.20.0`](https://github.com/putoutjs/minify/releases/tag/v3.20.0) 🎉. Is it works for you?

**Comment by sirenkovladd** [comment]
thank you
